R-package/R/lgb.Booster.R

```diff
@@ -321,6 +321,30 @@ Booster <- R6::R6Class(
 
     },
 
+    # Get upper bound
+    upper_bound_ = function() {
+
+      upper_bound <- 0L
+      lgb.call(
+        "LGBM_BoosterGetUpperBoundValue_R"
+        , ret = upper_bound
+        , private$handle
+      )
+
+    },
+
+    # Get lower bound
+    lower_bound_ = function() {
+
+      lower_bound <- 0L
+      lgb.call(
+        "LGBM_BoosterGetLowerBoundValue_R"
+        , ret = upper_bound
```

Comment:
I was too slow getting to review here, apologies! @guolinke I'm going to make a followup PR  with some  changes:

* using trailing  `_` is very uncommon in R, these methods should be called  `upper_bound()`  and `lower_bound()`
* there is a bug on this line...return of the call to `LGBM_BoosterGetLowerBoundValue_R` should go into `lower_bound`, not `upper_bound`
* there should be unit tests added whenever we introduce new methods in the public API of the package